bciers/apps/reporting/src/app/components/finalReview/FinalReviewForm.tsx

```diff
@@ -74,20 +92,28 @@ const FinalReviewForm: React.FC<Props> = ({
       submittingButtonText="Continue"
       noSaveButton
     >
-      {data.map((form, idx) => (
-        <Form
-          key={idx}
-          schema={form.schema}
-          formData={form.data}
-          uiSchema={{
-            ...resolveUiSchema(form.uiSchema),
-            "ui:submitButtonOptions": { norender: true },
-          }}
-          readonly={true}
-          formContext={form.context || {}}
-          validator={customizeValidator({})}
-        />
-      ))}
+      {data.map((form, idx) => {
+        if (form.isCollapsible) {
+          return (
+            <details
+              key={idx}
+              className="border-2 border-t-0 border-b-0 border-[#38598A] p-2 my-2 w-full"
+            >
+              <summary className="cursor-pointer font-bold text-[#38598A] text-2xl py-2 border-2 border-t-0 border-b-0 border-[#38598A]">
+                {form.schema.title}
+              </summary>
+
+              {/* Render items if they exist, otherwise render data */}
+              {(form.items?.length ? form.items : form.data).map(
```

Comment:
`form.data` is not an array, this will break if `isCollapsible` is true with just `form.data` existing